docker-compose.yml

```diff
@@ -1,6 +1,44 @@
 version: '3.7'
 
 services:
+  zookeeper:
+    container_name: zookeeper
+    image: docker.io/bitnami/zookeeper:3.6
+    ports:
+      - "2181:2181"
+    volumes:
+      - "zookeeper_data:/bitnami"
+    env_file:
+      - ./conf/local/zookeeper.env
+  kafka:
+    container_name: kafka
+    image: docker.io/bitnami/kafka:3.1
+    ports:
+      - "9092:9092"
+    volumes:
+      - "kafka_data:/bitnami"
+    env_file:
+      - ./conf/local/kafka.env
+    depends_on:
+      - zookeeper
+
+  kafka_init_topics:
+    image: docker.io/bitnami/kafka:3.1
+    depends_on:
+      - kafka
+    entrypoint: [ '/bin/sh', '-c' ]
+    command: |
+      "
+      # blocks until kafka is reachable
+      kafka-topics.sh --bootstrap-server kafka:9092 --list
+
+      echo -e 'Creating kafka topics'
+      kafka-topics.sh --bootstrap-server kafka:9092 --create --if-not-exists --topic test_sha --replication-factor 1 --partitions 1
```

Comment:
I'd probably use the topic name which will be used in real deployment instead of `test_sha`, but this is a minor thing.